src/main/java/ladder/domain/LinePoint.java

```diff
@@ -0,0 +1,26 @@
+package ladder.domain;
+
+public class LinePoint {
+    private final int position;
+    private final Point point;
+
+    public LinePoint(int position, Point point) {
+        this.position = position;
+        this.point = point;
+    }
+
+    public int move() {
+        Direction direction = point.move();
+        if (direction == Direction.LEFT) {
+            return position - 1;
+        }
+        if (direction == Direction.RIGHT) {
+            return position + 1;
+        }
+        return position;
+    }
```

Comment:
Enum 을 객체로 활용해보는것도 좋을것 같아요!

```java
public enum Direction {
    LEFT(-1),
    RIGHT(1),
    SOUTH(0);

    private final int offset;

    Direction(int offset) {
        this.offset = offset;
    }

    public int move(int position) {
        return position + offset;
    }
}
```

```suggestion
    public int move() {
        Direction direction = point.move();
        return direction.move(position);
    }
```